Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/client.py", line 1107, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/client.py", line 852, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from pm4py.visualization.heuristics_net import visualizer as hn_visualizer
gviz = hn_visualizer.apply(heu_net)
hn_visualizer.view(gviz)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mInvocationException[0m                       Traceback (most recent call last)
[0;32m<ipython-input-5-1910b53edf96>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mfrom[0m [0mpm4py[0m[0;34m.[0m[0mvisualization[0m[0;34m.[0m[0mheuristics_net[0m [0;32mimport[0m [0mvisualizer[0m [0;32mas[0m [0mhn_visualizer[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mgviz[0m [0;34m=[0m [0mhn_visualizer[0m[0;34m.[0m[0mapply[0m[0;34m([0m[0mheu_net[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0mhn_visualizer[0m[0;34m.[0m[0mview[0m[0;34m([0m[0mgviz[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/pm4py/visualization/heuristics_net/visualizer.py[0m in [0;36mapply[0;34m(heu_net, parameters, variant)[0m
[1;32m     35[0m         [0mRepresentation[0m [0mof[0m [0mthe[0m [0mHeuristics[0m [0mNet[0m[0;34m[0m[0;34m[0m[0m
[1;32m     36[0m     """
[0;32m---> 37[0;31m     [0;32mreturn[0m [0mexec_utils[0m[0;34m.[0m[0mget_variant[0m[0;34m([0m[0mvariant[0m[0;34m)[0m[0;34m.[0m[0mapply[0m[0;34m([0m[0mheu_net[0m[0;34m,[0m [0mparameters[0m[0;34m=[0m[0mparameters[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     38[0m [0;34m[0m[0m
[1;32m     39[0m [0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/pm4py/visualization/heuristics_net/variants/pydotplus.py[0m in [0;36mapply[0;34m(heu_net, parameters)[0m
[1;32m    249[0m     [0mfile_name[0m [0;34m=[0m [0mtempfile[0m[0;34m.[0m[0mNamedTemporaryFile[0m[0;34m([0m[0msuffix[0m[0;34m=[0m[0;34m'.'[0m [0;34m+[0m [0mimage_format[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    250[0m     [0mfile_name[0m[0;34m.[0m[0mclose[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 251[0;31m     [0mgraph[0m[0;34m.[0m[0mwrite[0m[0;34m([0m[0mfile_name[0m[0;34m.[0m[0mname[0m[0;34m,[0m [0mformat[0m[0;34m=[0m[0mimage_format[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    252[0m     [0;32mreturn[0m [0mfile_name[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/pydotplus/graphviz.py[0m in [0;36mwrite[0;34m(self, path, prog, format)[0m
[1;32m   1916[0m [0;34m[0m[0m
[1;32m   1917[0m             [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1918[0;31m                 [0mfobj[0m[0;34m.[0m[0mwrite[0m[0;34m([0m[0mself[0m[0;34m.[0m[0mcreate[0m[0;34m([0m[0mprog[0m[0;34m,[0m [0mformat[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1919[0m         [0;32mfinally[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1920[0m             [0;32mif[0m [0mclose[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/pydotplus/graphviz.py[0m in [0;36mcreate[0;34m(self, prog, format)[0m
[1;32m   1958[0m             [0;32mif[0m [0mself[0m[0;34m.[0m[0mprogs[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1959[0m                 raise InvocationException(
[0;32m-> 1960[0;31m                     'GraphViz\'s executables not found')
[0m[1;32m   1961[0m [0;34m[0m[0m
[1;32m   1962[0m         [0;32mif[0m [0mprog[0m [0;32mnot[0m [0;32min[0m [0mself[0m[0;34m.[0m[0mprogs[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;31mInvocationException[0m: GraphViz's executables not found
InvocationException: GraphViz's executables not found

